x-pack/test/saved_object_api_integration/security_and_spaces/apis/import.ts

```diff
@@ -53,27 +104,77 @@ export default function ({ getService }: FtrProviderContext) {
     esArchiver,
     supertest
   );
-  const createTests = (spaceId: string) => {
-    const { importableTypes, nonImportableTypes, allTypes } = createTestCases(spaceId);
-    // use singleRequest to reduce execution time and/or test combined cases
+  const createTests = (overwrite: boolean, trueCopy: boolean, spaceId: string) => {
+    const singleRequest = true;
+
+    if (trueCopy) {
+      const { importable, nonImportable, all } = createTrueCopyTestCases();
+      return {
+        unauthorized: [
+          createTestDefinitions(importable, true, { trueCopy, spaceId }),
+          createTestDefinitions(nonImportable, false, { trueCopy, spaceId, singleRequest }),
+          createTestDefinitions(all, true, {
+            trueCopy,
+            spaceId,
+            singleRequest,
+            responseBodyOverride: expectForbidden('bulk_create')([
+              'dashboard',
+              'globaltype',
+              'isolatedtype',
+              'sharedtype',
+            ]),
+          }),
+        ].flat(),
+        authorized: createTestDefinitions(all, false, { trueCopy, spaceId, singleRequest }),
+      };
+    }
+
+    const {
+      group1Importable,
```

Comment:
The names of these groups aren't all that clear to me. What does each group represent?

Reply:
Well, the groups of requests are split out into as few batches as possible to achieve the appropriate test coverage. At the time it didn't seem like it was worth coming up with more descriptive names (the names would be really long?), but:

* group 1 importable: results in 200 or 403 error (depending on authZ). we submit each test case separately because we get different 403 error messages depending on which type is present.
* group 1 non importable: results in 400 error always
* group 1 all: results in a 200 or 403 (but the 403 doesn't have the non-importable type in the error message)
* group 2: separate group for multi-namespace object types. A few of these could actually be moved into group 1, but some of them need to be created in a single request to trigger the appropriate conflict scenarios.
* group 3: another group of multi-namespace objects to trigger a different conflict scenario.
* group 4: another group of multi-namespace objects to trigger a different conflict scenario.

I could have created several more test data cases with different objects (e.g., `CASES.CONFLICT_5A_OBJ`, `CASES.CONFLICT_5B_OBJ`, etc.) but it seemed silly to do that when we already have all of the different object / origin combinations present in the existing test data to test our different conflict scenarios.